### PR TITLE
EWCODE-58 update styling to improve text vs metadata readability

### PR DESCRIPTION
**Pull request by jdpdev**

### CODE REVIEW

**Review by dsteedRTI — CHANGES_REQUESTED** [review]
Hey, Jason, really like the look of this! You did a great job!

The code is good, we just need to replicate it across the site. I'm not sure if you're aware, but the cards all are different components between the Annotate, History, Skew, IRR, and Recycle Bin tabs... So we should probably replicate this across all of those cards. Does that make sense?

For example, here's the History tab:
Screen Shot 2021-09-23 at 3 49 02 PM

**Comment by jdpdev** [comment]
@dsteedRTI Refactored all of the tabs to use a common `AnnotateCard` component

**Comment by dsteedRTI** [comment]
Looks awesome! I'm gonna approve, but let @AstridKery also take a look

**Review by dsteedRTI — APPROVED** [review]

File: frontend/src/components/AdminTable/index.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {
 } from "react-bootstrap";
 import Select from "react-dropdown-select";
 import CodebookLabelMenuContainer from "../../containers/codebookLabelMenu_container";
+import AnnotateCard, { buildCard } from "../AnnotateCard";
```
Comment (AstridKery) [inline frontend/src/components/AdminTable/index.jsx:12]: Looks great! My only issue is I noticed we are getting some linting errors from the AdminTable and Skew modules. I'm not sure when these were introduced, but we should probably clear them out. 

To reproduce the errors, run `docker-compose -f envs/dev/docker-compose.yml run --rm smart_frontend yarn lint`

```
/code/src/components/AdminTable/index.jsx
   5:5  warning  'Button' is defined but never used          no-unused-vars
   6:5  warning  'ButtonToolbar' is defined but never used   no-unused-vars
   7:5  warning  'Tooltip' is defined but never used         no-unused-vars
   8:5  warning  'OverlayTrigger' is defined but never used  no-unused-vars
  10:8  warning  'Select' is defined but never used          no-unused-vars

/code/src/components/Skew/index.jsx
  63:13  warning  'labelsOptions' is assigned a value but never used  no-unused-vars

✖ 6 problems (0 errors, 6 warnings)
```

Comment (jdpdev) [reply frontend/src/components/AdminTable/index.jsx:5]: fixed

Comment (AstridKery) [reply frontend/src/components/AdminTable/index.jsx:5]: Great. Thanks!

**Pull request merged**